src/app/loans/loans.service.ts

```diff
@@ -59,6 +59,39 @@ export class LoansService {
     return this.http.get(`/loans/${loanId}`, { params: httpParams });
   }
 
+  /**
+   * Approve Loan.
+   * @param {string} loanId Loan Id.
+   * @param {any} loanData Loan Data.
+   * @returns {Observable<any>}
+   */
+  approveLoan(loanId: string, loanData: any): Observable<any> {
+    const httpParams = new HttpParams()
+    .set('command', 'approve');
+    return this.http.post(`/loans/${loanId}`, loanData, {params: httpParams});
+  }
+  /**
+   * Get Multi Disburse Loan details.
+   * @param {string} loanId Loan Id.
+   * @returns {Observable<any>}
+   */
+  getMultiDisburseLoanDetails(loanId: string): Observable<any> {
+    const httpParams = new HttpParams()
+    .set('associations', 'multiDisburseDetails');
+    return this.http.get(`/loans/${loanId}`, {params: httpParams});
+  }
```

Comment:
@BLasan I don't find this function been used anywhere. Can you please explain its purpose?

Reply:
Sorry this is not needed. I'll remove